### PR TITLE
190701229_calculator_c++

### PR DESCRIPTION
**Pull request by thuhinkhanna**
Function Definitions of Arithmetic Operations are Updated.

### CODE REVIEW

File: calculator.cpp [hunk 1]
```diff
@@ -2,26 +2,32 @@
...
 }
 
 int multiplication(int number_1, int number_2){
-
+return number_1*number_2;
 }
 
 int division(int number_1, int number_2){
-
+return number_1/number_2;
```
Comment (surabhiojha12) [inline calculator.cpp:17]: what if divider is 0? 5/0?

File: calculator.cpp [hunk 2]
```diff
@@ -2,26 +2,32 @@
...
 }
 
 int get_input(){
-
+int a,b;
+cout<<"\n Enter two Integers :: ";
+cin>>a>>b;
+cout<<"\n Sum of Numbers : "<<addition(a,b);
```
Comment (surabhiojha12) [inline calculator.cpp]: Displaying the numbers as well will improve user experience.
Sum of Numbers 6 and 5 is 11. Something like this.

Comment (thuhinkhanna) [reply calculator.cpp]: Thanks for the review. Will do the updates.

File: calculator.cpp [hunk 3]
```diff
@@ -2,26 +2,35 @@
...
 }
 
 int division(int number_1, int number_2){
-
+return number_1/number_2;
 }
 
 int get_input(){
-
+int a,b;
```
Comment (surabhiojha12) [inline calculator.cpp]: name the variables properly.

**Review by thuhinkhanna — COMMENTED** [review]
Accidentally Deleted calculator.cpp
pushing calculator.cpp AGAIN

**Pull request closed without merging**